src/background/browser-adapter.ts

```diff
@@ -94,7 +94,14 @@ export class ChromeAdapter extends ClientChromeAdapter implements BrowserAdapter
     }
 
     public getTab(tabId: number, callback: (tab: chrome.tabs.Tab) => void): void {
-        chrome.tabs.get(tabId, callback);
+        chrome.tabs.get(tabId, tab => {
+            if (tab === null) {
+                console.log(`Cannot find tab with id ${tabId}`);
```

Comment:
Should we be using the `Logger` class?

https://github.com/Microsoft/accessibility-insights-web/blob/master/src/common/logging/logger.ts

Reply:
chrome will log the error

Reply:
I get that `chrome` will log the errors but since we have wrapped all our logs using `Logger` class; does it make sense to be consistent here as well and use that same class?